.circleci/config.yml

```diff
@@ -7,6 +7,16 @@ executors:
         environment:
           MAVEN_OPTS: -Xmx2g
 
+  # Note: Need to investigate how we can use our own image and have it configured so we can run docker inside docker
```

Comment:
Just as hints, here: It would require a) Docker installed in our own build image and b) running `- setup_remote_docker` as the first step making the docker environment available. We did this [here](https://github.com/OpenNMS/opennms-helm/blob/develop/.circleci/config.yml#L102) we use this here, the difference is we use the official Docker image instead of our own.

Reply:
Does `setup_remote_docker` make the containers spawned by the test container siblings instead of children? Since our test container ends up spawning multiple containers, I think the desired behaviour is that they run on the host as sibling containers rather than as child containers inside.